test/performance-tests.sh

```diff
@@ -17,12 +17,14 @@
 # This script runs the performance tests; It is run by prow daily.
 # For convenience, it can also be executed manually.
 
+INSTALL_MONITORING=1
```

Comment:
This breaks the idea of enabling the feature by a flag. If you want to force it when calling install_knative_serving explicitly, better enforcing that in the function itself. Leave the flag for the default case only.

Reply:
Removed.